participation.html

```diff
@@ -13,7 +13,13 @@
     <div class="banner white">
       <div class="container">
         <h3 class="size8">Scout/Parent Participation</h3>
-        <p>Scout and parent participation are crucial to the success of Troop 370.  Learning, leadership, and advancement opportunities abound for Scouts.<br><br>For parents, there are many ways in which you can assist through volunteer efforts.  There is no greater joy than joining your Scout through their Troop 370 experience, and working to support the Troop "behind the scenes".  If you would like to see how you can help, please contact the Troop, attend a meeting, or check out the "What Can I Do?" link in the password protected members section of the site.  Meetings are held most Wednesday evenings from 7:30 - 9:00PM in the St. James UMC activities building.<br><br>Using the drop-down menu under "Scout/Parent Info" above, you will find forms, helpful information, and other common documents that you will find helpful during your journey through Scouts at Troop 370.</p>
+        <p>Scout and parent participation are crucial to the success of Troop 370.  Learning, leadership, and advancement opportunities abound for Scouts.</p>
+        <p>At Troop 370, there are monthly weekend campouts that the troop takes. Also, every summer, the Troop visits Woodruff Scout Reservation in Blairsville, GA. Recently, they have also been visiting the Summit Bechtel Reserve in West Virginia as a secondary summer camp. The Troop has a multitude of high adventure trips as well. They visit all high adventure bases (more info on high adventure <href> here). The troop also has many service projects open to scout participation.</p>
+        <p>Troop 370 is a boy-led troop, meaning that while the adult scouters perform administrative duties, the scouts plan activities and campouts. Scouts have the opportunity to take up leadership positions in the troop. While leadership positions are required for advancement, the level of the position is mostly optional. There are a multitude of leadership positions available for scouts to hold. Other than the required leadership for advancement, the level of scout participation for leadership is up to him.</p>
```

Comment:
```suggestion
        <p>Troop 370 is a boy-led troop, meaning that the scouts lead the meetings and plan the activities and campouts - not the adults. All scouts have the opportunity to take leadership positions in the troop, such as being a Senior Patrol Leader, Assistant Senior Patrol Leader, Instructor, Troop Guide, Patrol Leader, Scribe, Quartermaster, or Historian. Scouts can take leadership positions at any time or for any purpose; they do no need to wait until elections or for when they need it for advancement. Read more about our leadership in our <a href="src/documents/Guidelines%20for%20Positions%20of%20Leadership.pdf">Guidelines for Positions of Leadership document</a></p>
```
This section is about the scouts - not the adults. It is already mentioned in a later paragraph that the adults also have roles. An added benefit or removing the mention of the adults in this paragraph is it makes the tone of the section more positive. 

Additionally, I rephrased the last few sentences into a more concise, specific, and informative set of sentences that also prompt the reader to read more information.